Sources/MapboxMaps/Annotations/PointAnnotation+ImageHandling.swift

```diff
@@ -22,114 +18,16 @@ extension PointAnnotationManager {
     func addImageToStyleIfNeeded(style: Style) {
         let pointAnnotationImages = Set(annotations.compactMap(\.image))
         for pointAnnotationImage in pointAnnotationImages {
-            do {
-                let image = style.image(withId: pointAnnotationImage.name)
-
-                if image == nil { // If image not found, then add the image to  the style
-                    switch pointAnnotationImage {
-
-                    case .default: // Add the default image if not added already
-                        try style.addImage(Self.defaultMarker,
-                                           id: pointAnnotationImage.name,
-                                           stretchX: [],
-                                           stretchY: [])
-
-                    case .custom(image: let image, name: let name): // Add this custom image
-                        try style.addImage(image,
-                                           id: name,
-                                           stretchX: [],
-                                           stretchY: [])
-                    }
+            // If the image is not found, add it to the style
+            if style.image(withId: pointAnnotationImage.name) == nil {
+                do {
+                    try style.addImage(pointAnnotationImage.image, id: pointAnnotationImage.name, stretchX: [], stretchY: [])
```

Comment:
Do we need handling for cap insets here?

Reply:
Seems like that would be a new feature. Maybe we would want to do it, but I'd have to give it more thought. Regardless, if folks need this, they could bypass `PointAnnotation.image` and use `iconImage` + style image APIs directly.